editor/src/components/text-editor/text-editor.tsx

```diff
@@ -337,8 +343,17 @@ const TextEditor = React.memo((props: TextEditorProps) => {
   )
 
   const onBlur = React.useCallback(() => {
-    dispatch([updateEditorMode(EditorModes.selectMode())])
-  }, [dispatch])
+    const content = myElement.current?.textContent
+    if (content != null && elementState != null && savedContentRef.current !== content) {
+      savedContentRef.current = content
+      dispatch([
+        updateChildText(elementPath, escapeHTML(content)),
```

Comment:
I think this updateChildText(elementPath, escapeHTML(content) is worth pulling into a function – just in case we need to dispatch it from a third place, to avoid someone not copy-pasting it

Reply:
done